openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/validation/RequestValidator.java

```diff
@@ -107,8 +129,33 @@ public OperationValidator getValidator(final Path path, final Operation operatio
    * @param request The request to validate. Must be {@code nonnull}.
    * @throws ValidationException A validation report containing validation errors
    */
-  public RequestParameters validate(final Request request) throws ValidationException {
-    return validate(request, new ValidationData<>());
+  public void validate(final Request request) throws ValidationException {
```

Comment:
Why did you remove the returned parameters?

Reply:
Sorry, that was a mistake, will revert